docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,373 @@
+</ScopedBlock>
+
+```code
+$ tctl get roles --format=text
+```
+
+You will see something like the output below on a fresh install of Teleport with the
+default roles—your cluster may have different roles. In this example, let's
+assume you want to give the bot the `hostcert-bot` role to allow it to generate host certificates.
+
+```
+Role    Allowed to login as                           Node Labels Access to resources
+------- --------------------------------------------- ----------- ----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read
+editor
+hostcert-bot  host_cert:create,list,read,update,delete             user:list,create,read,update,delete,...
+```
+
+`tbot` can join the Teleport Cluster using a number of supported methods. This guide will highlight both the `token` and the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
+Note that the following commands may be entered on any Teleport node with `tctl` access to the cluster.
+
+While any role with permissions for creating host certificates will work, this guide will make use of the `hostcert-bot` role created previously to ensure that the configuration adheres to the principle of least privilege.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+
+  ```code
+  $ tctl bots add robot --roles=hostcert-bot
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  First, create an IAM method token that specifies the AWS account from which
+  the bot can join. Create the below file as `iam-token.yaml` then run `tctl
+  create -f iam-token.yaml`.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time for how long you want to support IAM method for
+    # joining. It is safe to set this value to a very long time.
+    expires: "3000-01-01T00:00:00Z"
+  spec:
+    # Only allow bots to join using this token.
+    roles: [Bot]
+
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token.
+    bot_name: robot
+
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=hostcert-bot
+  ```
+  </TabItem>
+</Tabs>
+
+Machine ID may now be configured using the `tbot configure` command on the bot node.
+
+First, create a basic configuration file using the following parameters:
```

Comment:
What does "First" mean here? This seems like an intermediate step within this section. Maybe we should start a separate H3 section here?